### ISSUE TITLE
IXLColumn.Delete() sometimes creates corrupt XLSX files.

### ISSUE THREAD

**Issue by albinsunnanbo** [ClosedXML/ClosedXML#205]
Reproducing example: https://github.com/albinsunnanbo/ClosedXmlColumnBug
To reproduce, just run the command line app and look in bin\debug\DoWork.xlsx and bin\debug\DoesntWork.xlsx

I create a table with `IXLCell.InsertTable` from a `List<T>`: https://github.com/albinsunnanbo/ClosedXmlColumnBug/blob/master/TestExcelExport/ExcelExporter.cs#L44

Then I remove a column from the worksheet with `IXLColumn.Delete`: https://github.com/albinsunnanbo/ClosedXmlColumnBug/blob/master/TestExcelExport/ExcelExtensions.cs#L38

If my `T` is DoWorkModel

    public class DoWorkModel
    {
        [Display(Name = "First string")]
        public string String1 { get; set; }

        [Display(Name = "Second string")]
        public string String2 { get; set; }

        [Display(Name = "The Int")]
        public int? Int1 { get; set; }

    }

And I remove the third column (Int1) a valid XLXS document is created.

On the other hand, if my `T` is DoesntWorkModel

    public class DoesntWorkModel
    {
        [Display(Name = "First string")]
        public string String1 { get; set; }

        [Display(Name = "The Int")]
        public int? Int1 { get; set; }

        [Display(Name = "Second string")]
        public string String2 { get; set; }
    }

And I remove the second column (Int1) a corrupt XLXS document is created.
When trying to open the document in Excel the following dialog is shown:
![image](https://cloud.githubusercontent.com/assets/5158396/22974700/dfbfbb68-f383-11e6-8b81-5b355204437c.png)
and then
![image](https://cloud.githubusercontent.com/assets/5158396/22974710/eb02eea0-f383-11e6-9d9c-ad3775fba39f.png)

and the resulting log file

	<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
	<recoveryLog xmlns="http://schemas.openxmlformats.org/spreadsheetml/2006/main">
		<logFileName>error324960_01.xml</logFileName>
		<summary>Errors were detected in file 'C:\Git\ClosedXmlColumnBug\TestExcelExport\bin\Debug\DoesntWork.xlsx'</summary>
		<repairedRecords summary="Following is a list of repairs:">
			<repairedRecord>Repaired Records: Table from /xl/tables/table.xml part (Table)</repairedRecord>
		</repairedRecords>
	</recoveryLog>

If I unzip the resulting xlsx files and run a diff compare i find an interesting difference in the table.xml file.
See diff below (table.xml pretty printed with line breaks for clarity)

![image](https://cloud.githubusercontent.com/assets/5158396/22974775/2dfd6e9c-f384-11e6-805f-618a239cbd35.png)
With `DoesntWorkModel` (to the left in the diff) `tableColumn` has the wrong name (that column was deleted).

It looks like ClosedXML doesn't correctly delete the correct tableColumn when deleting columns from the workSheet

**Comment by igitur** [comment]
Yes, an ignore attribute would be good. I'll do that first and then look at the rest of the bug later.

**Comment by albinsunnanbo** [comment]
Thanks for super fast response. The ignore attribute would be much better than my ugly hack of removing unwanted columns after generation and avoid the initial problem in my case.

**Comment by igitur** [comment]
Check #209  for new `XLColumnAttribute`.
